### DISCUSSION TITLE
Is it possible to have warmup cycles before rendering with graphin-force?

### DISCUSSION THREAD

**Discussion by albcunha** [antvis/Graphin#469] [Q&A] [answered]
When I load my graph, it takes some time for the graph to conform using graphin-force.

Is it possible to make these calculations without rendering, so It would be faster? Some other libraries offer this feature, such as react-force-graph (https://github.com/vasturiano/react-force-graph), with this property:

<html>
<body>

warmupTicks | number | 0 | Number of layout engine cycles to dry-run at ignition before starting to render.
-- | -- | -- | --

</body>
</html>

**Comment by pomelo-nwu** [comment, marked as answer]
Three Options: 
1. Turn off animation 
3. Use force2 Layout 
3. nodes position is calculated in advance, and the layout type is set to preset
